tests/debian_test.yaml

```diff
@@ -1,4 +1,14 @@
 schemaVersion: '2.0.0' # Make sure to test the latest schema version
+
+driver: "docker"
+
+containerRunOpts:
+  bindMounts:
+    - "/tmp/test:/tmp/test"
```

Comment:
I've actually been thinking about this overnight...

I'm wondering whether we should support environment variable interpolation here. Because Docker requires absolute paths. So being able to interpolate a `~` or `$PWD` would be really useful.

Reply:
Hmm, yeah I see what you mean. In order to do that we'd need to first parse the config, then somehow retrieve the env from the image, and then substitute that into a new config before running the tests. This is probably doable, but might get tricky especially if we're not using docker. Feel free and give it a shot though, I'd like to see what it looks like